dear_petition/petition/export/tests/test_cr287_3b.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+
+from django.utils import timezone
+
+from dear_petition.petition.tests.factories import (
+    CIPRSRecordFactory,
+    OffenseFactory,
+    OffenseRecordFactory,
+    PetitionFactory,
+)
+from dear_petition.petition.types import dismissed
+from dear_petition.petition import constants
+from dear_petition.petition.export.forms import AOCFormCR287
+
+pytestmark = pytest.mark.django_db
+
+
+def create_record(batch, jurisdiction, county):
+    record = CIPRSRecordFactory(
+        batch=batch,
+        label=batch.label,
+        jurisdiction=jurisdiction,
+        county=county,
+    )
+    guilty_offense = OffenseRecordFactory(
+        action="CONVICTED",
+        description="BREAK OR ENTER A MOTOR VEHICLE",
+        severity="FELONY",
+        offense=OffenseFactory(
+            verdict="GUILTY",
+            disposition_method="DISPOSED BY JUDGE",
+            disposed_on=timezone.now(),
```

Comment:
This is a fringe edge case, but with multiple calls to `timezone.now()` there's a very narrow window for the offense to be on separate days, like if we run the testcase milliseconds before midnight. I'll leave it up to you, but we could make a `time_disposed` variable that stores `timezone.now()` and pass it as the `disposed_on` value for each offense and offense record. Not a high priority change, so up to your discretion.